packages/repmgr13/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM registry.access.redhat.com/ubi9/ubi
+
+ENV TERM=xterm \
+    RPM_SPEC=repmgr.spec
+
+RUN ARCH=$(uname -m) && \
+    # TODO figure out s390x
+    dnf -y remove *subscription-manager* && \
+    dnf -y update && \
+    dnf -y install \
+      http://mirror.stream.centos.org/9-stream/BaseOS/${ARCH}/os/Packages/centos-stream-repos-9.0-18.el9.noarch.rpm \
+      http://mirror.stream.centos.org/9-stream/BaseOS/${ARCH}/os/Packages/centos-gpg-keys-9.0-18.el9.noarch.rpm \
+      https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm && \
+    dnf -y group install "development tools" && \
+    dnf -y install --enablerepo=crb \
```

Comment:
What is `crb`?

Reply:
the contrib repository 

contrib now has postgresql-server-devel and postgresql-static so must be enabled

Reply:
the latest commit explains it

No need for postgresql modules since pg 13 is the default in stream 9

Changes to get ubi9/stream9 working:

1) pgxs.mk is needed and provided by postgresql-server-devel
2) libpgcommon.a and libpgport.a are needed and provided by postgresql-static
3) Contrib repo now contains the postgresql-server-devel and postgresql-static
packages so we need to enable crb
4) tmpfiles_create macro is either removed/deprecated and suggested to use
tmpfiles_create_package with the newer systemd in ubi9/stream9.

Note, postgresql-server-devel requires postgresql-private-devel which also
provides libpq and so conflicts with libpq-devel.  postgresql-static also
requires postgresql-server-devel.

libpgcommon.a and libpgport.a were excluded form devel packages and instead put
into the postgresql-static package and discussed in a prior commit.  See this bz
for more information:  https://bugzilla.redhat.com/show_bug.cgi?id=784281

Reply:
This conversation was related to ubi9 which is no longer in this PR.  Resolving this conversation.